packages/app-editors/kakoune/kakoune.exlib

```diff
@@ -0,0 +1,63 @@
+# Copyright 2014 Maxime Coste <frrrwww@gmail.com>
+# Distributed under the terms of the GNU General Public License v2
+
+require toolchain-funcs github [ user=mawww tag=v${PV} ]
```

Comment:
I think there is some utf8-locale exlib that could be usefull here to ensure tests will pass (some tests do requires an utf8 locale)

Reply:
Fixed (tests pass).